packages/ui-components/style/base.css

```diff
@@ -145,7 +145,7 @@ input::placeholder {
 
 .jp-InputGroupAction {
   position: absolute;
-  top: 0;
+  bottom:1px;
```

Comment:
It seems you create your branch on top of the one for the other PR.

Let's remove this change for the sake of separation of concern.

```suggestion
  top: 0;
```